### ISSUE TITLE
Standalone package based on this library

### ISSUE THREAD

**Issue by jan-j** [darsain/laravel-console#40]
Hi
I just wanted to notify that some time ago I forked this package and created standalone alternative (standalone means that can be used with any framework or with vanilla PHP). The general functionality and appearance of this application stayed the same, but I improved a lot and added some other functionality. I just released version 1.0.0.

Link to repository: https://github.com/meebio/php-eval-console

**Comment by darsain** [comment]
Sure, no problem. But the front-end of this package was a bloated jQuery mess that I never got to rewrite. I feel weird seeing it forked and spreading :disappointed: ohwell.

**Comment by tortuetorche** [comment]
@jan-j Do you plan to add the Laravel `config.php` file, to keep some backward compatibility with `laravel-console`? Maybe with a new package.

**Comment by jan-j** [comment]
@darsain If I remember correctly I fixed some issues with JS scripts (it was long ago so I don't recall what it was). There probably should be option to set max timeout (both for ajax request and php evaluation), but other than that I don't think it is worth to rewrite it. It's working fine.
The biggest thing I did with scripts and stylesheets is setting up gulpfile that is concatenating and minifying them.

@tortuetorche I can probably add some section in docs about integrating with Laravel and maybe special provider for it to create config and copy assets. It's really easy to setup it with only one route entry, but I guess it doesn't hurt to make it more Laravel way.
